splice_access_api/src/main/java/com/splicemachine/access/configuration/StorageConfiguration.java

```diff
@@ -36,10 +36,13 @@
     public static final String SPLIT_BLOCK_SIZE = "splice.splitBlockSize";
     public static final int DEFAULT_SPLIT_BLOCK_SIZE=32*1024*1024;
 
+    public static final String SPLITS_PER_TABLE_MINIMUM = "splice.splitsPerTableMinimum";
```

Comment:
What do you think about a change to have the names like `MINIMUM_SPLITS_PER_TABLE` and `splice.minimumSplitsPerTable` (minimum is a prefix not a suffix).